dist/js/bootstrap.js

```diff
@@ -8,9 +8,9 @@ if (typeof jQuery === 'undefined') {
   throw new Error('Bootstrap\'s JavaScript requires jQuery')
 }
 
-+function ($) {
+function ($) {
```

Comment:
See http://stackoverflow.com/questions/13341698/javascript-plus-sign-in-front-of-function-name
@fat prefers this instead of using a wrapping pair of parentheses.